jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/VuetifyButtonGroupsTests.java

```diff
@@ -39,13 +36,12 @@ public void mandatoryButtonGroupTest() {
         mandatoryButtonGroup.is().displayed();
         mandatoryButtonGroup.has().css("width", "197px");
         mandatoryButtonGroup.getButtonByIndex(1).has().css("width", "50px");
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(1));
-        mandatoryButtonGroup.getButtonByIndex(2).click();
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(2));
 
-        mandatoryButtonGroup.getAllButtons().forEach(HasClick::click);
-
-        assertSelected(mandatoryButtonGroup.getButtonByIndex(4));
+        mandatoryButtonGroup.is().selected(1);
+        mandatoryButtonGroup.getButtonByIndex(2).click();
```

Comment:
There is no checks for getButtonByText method, should be added to improve readability

Reply:
For these checks another task was created https://github.com/jdi-testing/jdi-light/issues/4107